dll/kernel32.cpp

```diff
@@ -10,15 +10,42 @@
 #include <sys/stat.h>
 
 namespace kernel32 {
+	char *WideStringToString(const uint16_t *src) {
```

Comment:
user32.cpp uses `static` + lowerCamelCase for local helpers:
```suggestion
	static char *wideStringToString(const uint16_t *src) {
```
Admittedly the `static` part of that doesn't make sense: all functions in the `kernel32` namespace deserve it. But I think some indicator like that that the function isn't a winapi function is useful.